### ISSUE TITLE
Rultor merge build broken

### ISSUE THREAD

**Issue by llorllale** [yegor256/cactoos#997]
See [error](https://github.com/yegor256/cactoos/pull/995#issuecomment-452941691).

Solution is to `sudo apt install python-pip` before `sudo pip install gitlint`.

PRs blocked by this: #995, #975

**Comment by llorllale** [comment]
@0crat in

**Comment by llorllale** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/yegor256/cactoos/issues/997#issuecomment-452945097))

@llorllale Job [#997](https://github.com/yegor256/cactoos/issues/997) is already in scope

**Comment by 0crat** [comment]
The job [#997](https://github.com/yegor256/cactoos/issues/997) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C63314D6Z/9fcd8db1-42ff-4d55-bb89-72957c377a6d); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); we should be aware that @llorllale[/z](https://www.0crat.com/u/llorllale) is on vacation; this ticket may be delayed; there will be no monetary reward for this job

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/cactoos/issues/997#issuecomment-452945137))

@llorllale Job `gh:yegor256/cactoos#997` already assigned to @llorllale, can't assign to @llorllale

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#997` already assigned to @llorllale, can't assign to @llorllale

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Job was finished in 11 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#997](https://github.com/yegor256/cactoos/issues/997) is now out of scope
